sql/pg_duckdb--0.2.0--0.3.0.sql

```diff
@@ -17,3 +17,226 @@ CREATE AGGREGATE @extschema@.approx_count_distinct(anyelement)
 
 CREATE DOMAIN pg_catalog.blob AS bytea;
 COMMENT ON DOMAIN pg_catalog.blob IS 'The DuckDB BLOB alias for BYTEA';
+
+-- json_exists
+CREATE FUNCTION @extschema@.json_exists("json" json , path VARCHAR)
+RETURNS boolean LANGUAGE 'plpgsql'
+SET search_path = pg_catalog, pg_temp
+AS
+$func$
+BEGIN
+    RAISE EXCEPTION 'Function `jsonb_exists(json, VARCHAR)` only works with Duckdb execution.';
```

Comment:
I just merged #531. Which introduces a more re-usable way of doing this:

```
AS 'MODULE_PATHNAME', 'duckdb_only_function'
LANGUAGE C;
```

Reply:
Fixed